### DISCUSSION TITLE
Negative impact of updates

### DISCUSSION THREAD

**Discussion by yuriks59** [flybywiresim/aircraft#5256] [General]
There is a general trend, as the update so something stops working, or it works incorrectly. After the last two updates, after turning on the power, the vectors themselves turn on, the door closing button no longer lights up, the navigation lights do not light up after turning on the batteries, which does not indicate that the crew is in the cockpit. Many buttons on the top panel are not highlighted until the external power is turned on or the APU starts.

**Comment by tracernz** [comment]
Unless you consider realism negative, these are positives.

**Comment by hotshotp** [comment]
everything that you mentioned other than the door closing button on the efb (which will be fixed with later flypad v2) is accurate and realistic. maybe try to research things before writing comments like these.
